### ISSUE TITLE
Check for updates.

### ISSUE THREAD

**Issue by htmlboss** [DFO-Ocean-Navigator/Navigator-Installer#8]
Since we have md5 files on Dory we can easily check for updates by comparing the md5 hashes, and performing an update if necessary.

**Comment by jmunroe** [comment]
Which files do we need to update?  Not sure it is clear which md5 files we are talking about here.

(If issue is stale, please close)

**Comment by dwayne-hart** [comment]
Myself and Nabil were discussing how we could have the system look for updates automatically in the event the Ocean Navigator were restarted. This is possible if we wanted to set an environment variable in the [...]/tools/conf/ocean-navigator-env.sh file to indicate that the person or group running an Ocean Navigator instance wish to keep up to date automatically. The default setting would be set to false so that people are able to opt in or out of this particular feature.

**Comment by jmunroe** [comment]
What does "an update" mean in this context? A pull from GitHub?

**Comment by dwayne-hart** [comment]
Yes, an update for this use case would be based on a pull from GitHub.
